pkg/kubelet/config/file_linux_test.go

```diff
@@ -262,7 +246,17 @@ func watchFileAdded(watchDir bool, t *testing.T) {
 				t.Fatalf("unable to create temp dir: %v", err)
 			}
 			defer os.RemoveAll(dirName)
+
 			fileName := fmt.Sprintf("%s_%d", fileNamePre, index)
+			var linkedDirName string
+			if isSymbol {
+				linkedDirName, err = utiltesting.MkTmpdir("linked-dir-test")
```

Comment:
I'm not a huge fan of relying on testing utilities that were written specifically for other packages and live in other repos, there's no guarantee they won't change all-of-a-sudden and break your tests.